src/components/NetworkSelector/index.tsx

```diff
@@ -40,7 +40,10 @@ const themes: Record<string, Object> = {
   mainnet: {
     color: '#0EAC1B',
   },
-  testnet: {
+  testnet1: {
+    color: '#FF7575',
```

Comment:
Should we use different color for testnet 2?

Reply:
Nah that's fine